.github/dependabot.yml

```diff
@@ -41,7 +41,7 @@ updates:
     directory: "/"
     schedule:
       interval: daily
-    target-branch: "release/v10.0.x"
+    target-branch: "release/v11.x"
```

Comment:
Should we add a comment that this is the last 3 releases, with a reference to e.g. https://github.com/cosmos/gaia/blob/main/RELEASING.md#major-release-maintenance?

Reply:
We should keep it as at the moment the latest two _in-production_ releases are v11 and v12.